src/java.base/share/classes/java/lang/Throwable.java

```diff
@@ -65,7 +65,7 @@
  * it is generally unrelated to the abstraction provided by the upper layer.
  * Further, doing so would tie the API of the upper layer to the details of
  * its implementation, assuming the lower layer's exception was a checked
- * exception.  Throwing a "wrapped exception" (i.e., an exception containing a
+ * exception.  Throwing a "wrapping exception" (i.e., an exception containing a
```

Comment:
Are we sure that this should be "wrapping" and not "wrapped?" See also for example `java.security.cert.CertPathValidatorException.`

Reply:
If we have two exceptions A and B, such that B is the cause of A, then A is the wrapping exception (the one that wraps or the wrapper) and B is the wrapped exception (the one that is being wrapped or the wrappee).

I noticed that the sentence was conflicting: it started with the "wrapped" exception, but then in the "i.e." commentary in parentheses proceeded with the wrappee exception. To resolve that conflict, I proposed to rephrase the first part of that sentence. FWIW, my original suggestion elsewhere was to use "wrapper".

Reply:
Subjectively, "wrapping exception" would seem to be an exception in the process of wrapping something.

Reply:
Would "wrappER" be better?

Reply:
Note that we don't throw the "wrapped exception" we throw the exception that wraps it. The "wrapped exception" is the original cause. The wording as presented now is correct in that regard. You could also say "Throwing a wrapper exception (i.e. one that has a cause)" - I think both are grammatically correct.

The later text "where a wrapped exception is thrown from the same method" is again incorrect because the wrapped exception (the cause) is not what gets thrown. But I find that whole sentence rather jarring anyway - I'm not certain exactly what it means.

Reply:
I'm inclined to revert this change on Throwable.java:68, because of the lack of consensus. It clearly is not a simple typo.

Reply:
Reverted in 57b71c5.